Makefile

```diff
@@ -28,6 +28,9 @@ bundle/camel-k: operator-sdk
 		"$(CAMEL_K_BUNDLE_NAME)" \
 		"$(CAMEL_K_BUNDLE_NAME)/$(ADDON_VERSION)" \
 		"stable"
+	
+	yq -i 'del(.spec.install.spec.deployments[].label)' \
```

Comment:
I don't see those those label fields in any of the CSVs. What is the intention here?

Reply:
mh, looks like those new labels are generated by newer operator-sdk version as they are added with 1.21.0 but not with 1.17.0

Reply:
I've commented out the processing step and added a TODO for future reference

Reply:
still there for camel-k and strimzi tho